03SetsDictionary/SetDictionaryProject/Tester.swift

```diff
@@ -8,37 +8,69 @@
 import Foundation
 
 struct Tester {
+    static let shared = Tester()
     /// Remove an array with the duplicated values removed
     func removeDups(nums: [Int]) -> [Int] {
-        
-        return []
+        return Array(Set(nums))
     }
     
     /// Find the nums that exist in both arrays
     func findCommonNums(from firstNums: [Int], and secondNums: [Int]) -> [Int] {
         
-        return []
+        return Array(Set(firstNums).intersection(Set(secondNums)))
     }
     
     /// first: "hello", second: "bye" -> ["e"]
     func findMatchingLetters(firstWord: String, secondWord: String) -> [Character] {
-        return []
+        return Array(Set(firstWord).intersection(Set(secondWord)))
     }
     
     
     /// Create a dictionary of the counts of the letters, ignoring spaces
     func createCounter(string: String) -> [Character: Int] {
-        return [:]
+        var dict: [Character:Int] = [:]
+        for char in string {
+            if(char != " "){
+                if(dict[char] != nil){
+                    dict[char]! += 1
+                }else{
+                    dict[char] = 1
+                }
+            }
+        }
+        return dict
     }
     
     /// Find most frequent letter in string: "hello there" -> "e"
     func mostFrequentLetter(string: String) -> Character {
```

Comment:
what is this? `toReturn = toReturn!`
can call `createCounter` to reduce code duplication